tests/cli/test_suite.py

```diff
@@ -840,7 +844,7 @@ def test_suite_new_interactive_valid_batch_request_from_json_file_in_notebook_ru
 
     assert mock_webbrowser.call_count == 0
 
-    assert mock_emit.call_count == 5
+    assert mock_emit.call_count == 4
```

Comment:
We are no longer saving the expectation suite (or adding a citation) before the notebook is loaded.